Hermes/appscale/hermes/stats/handlers.py

```diff
@@ -17,10 +17,25 @@ class CurrentStatsHandler(RequestHandler):
   """ Handler for getting current local stats of specific kind.
   """
 
-  def initialize(self, source, default_include_lists):
+  def initialize(self, source, default_include_lists, cache_container):
+    """ Initializes RequestHandler for handling a single request.
+
+    Args:
+      source: an object with method get_current.
+      default_include_lists: an instance of IncludeLists to use as default.
+      cache_container: a list containing a single element - cached snapshot.
+    """
     self._stats_source = source
     self._default_include_lists = default_include_lists
-    self._snapshot = None
+    self._cache_container = cache_container
+
+  @property
+  def _cached_snapshot(self):
+    return self._cache_container[0]
```

Comment:
Why is self._cache_container a list?

Reply:
RequestHandler instance is created for every single request, so cache container should be passed to initialize method.
I was thinking about defining dedicated class with a single attribute, but it looks like over-engineering. So list object seems to be the simplest solution.